### PR TITLE
Addition of EthersIntmaxWalletSigner

### PR DESCRIPTION
**Pull request by inaridiy**
## Describe your changes
EthersIntmaxWalletSigner is implemented.

## Issue ticket number and link
https://github.com/InternetMaximalism/webmax.js/issues/41

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [x] Will this be part of a product update? If yes, please write one phrase about this update.

Now, the webmax.js development experience takes a step further: EthersIntmaxWalletSigner.
Integrate ethers.js and webmax.js!

### CODE REVIEW

File: src/ethers-signer.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  }
+
+  async sendTransaction(
+    transaction: ethers.providers.TransactionRequest
+  ): Promise<ethers.providers.TransactionResponse> {
+    const result = (await this._intmaxWalletSigner.sendTransaction(
+      this.mapTransaction(transaction),
+      false
+    )) as unknown as Transaction;
+    console.log(result);
```
Comment (spiderings) [inline src/ethers-signer.ts:76]: Could you remove console.log?

**Comment by spiderings** [comment]
I tried it in the demo application. It works well. Thank you.

File: src/ethers-signer.ts (hunk 1 above)
Comment (inaridiy) [reply src/ethers-signer.ts:76]: Sorry! I'll turn it off right now.

Comment (spiderings) [reply src/ethers-signer.ts]: Thank you!

**Review by spiderings — APPROVED** [review]

**Pull request merged**